### PR TITLE
Use json_annotation, include copyright in generated files

### PR DESCRIPTION
**Pull request by kevmoo**
And also ignore prefer_final_locals

### CODE REVIEW

File: app/lib/shared/configuration.dart [hunk 1]
```diff
@@ -91,12 +91,13 @@ class EnvConfig {
...
   EnvConfig._(
-    this.gaeService,
+    String gaeService,
     this.gcloudProject,
     this.gcloudKey,
     this.flutterSdkDir,
     this.isolateCount,
-  ) {
+  )
+      : this.gaeService = gaeService ?? 'default' {
```
Comment (isoos) [inline app/lib/shared/configuration.dart]: I think @mkustermann asked me not to have a default value here and we keep being explicit about what service we want to start.

Comment (mkustermann) [reply app/lib/shared/configuration.dart]: @kevmoo This is an unrelated change (again), this change should do only one thing, not several different orthogonal things :-/

File: app/test/frontend/handlers_test_utils.dart [hunk 2]
```diff
@@ -177,12 +177,8 @@ class TemplateMock implements TemplateService {
...
 
   @override
-  String renderPkgIndexPage(
-      List<PackageView> packages,
-      PageLinks links,
-      {String title,
-      String faviconUrl,
-      String descriptionHtml}) {
+  String renderPkgIndexPage(List<PackageView> packages, PageLinks links,
+      {String title, String faviconUrl, String descriptionHtml}) {
```
Comment (mkustermann) [inline app/test/frontend/handlers_test_utils.dart]: ditto

File: app/test/frontend/handlers_test.dart [hunk 3]
```diff
@@ -252,8 +252,7 @@ void main() {
           return [testPackageVersion];
         });
         registerBackend(backend);
-        await expectAtomXmlResponse(await issueGet('/feed.atom'),
-            regexp: '''
+        await expectAtomXmlResponse(await issueGet('/feed.atom'), regexp: '''
```
Comment (mkustermann) [inline app/test/frontend/handlers_test.dart]: ditto.

**Comment by kevmoo** [comment]
PTAL – sticking w/ one change 😄

**Pull request merged**